nft_ingester/src/api/dapi/search_assets.rs

```diff
@@ -55,9 +55,6 @@ pub async fn search_assets<
     tree_gaps_checker: &Option<Arc<PPC>>,
     native_mint_pubkey: &str,
 ) -> Result<AssetList, StorageError> {
-    if options.show_fungible {
```

Comment:
wdyt of instead of removing this if to make the condition more precise:
```
if options.show_fungible &&  filter.token_type != Some(TokenType::Fungible) {
```
This will ensure we don't overwrite the request when the user asks specifically for fungibles

Reply:
This action looks very strange and not obvious, since in the middle of the function execution we change the user request option.

Reply:
in the current implementation this option _token_type_ / _show_fungible_ can be specified in the request, and I don't quite understand why do we need to change it in the process.